src/components/MenuPanel/MenuPanel.js

```diff
@@ -111,7 +111,7 @@ class MenuPanel extends React.PureComponent {
             style={{
               openProgress: spring(
                 Number(notificationsOpened),
-                springConf('fast')
+                springConf['fast']
```

Comment:
This probably returns `undefined`: to get the new spring configuration object, [`springs`](https://github.com/aragon/aragon-ui/blob/master/src/utils/styles/spring.js#L6) (with an `s`) should be imported instead of `spring`, making it unnecessary to rename it `springConf`.